src/main/java/at/hannibal2/skyhanni/config/features/combat/MobsConfig.java

```diff
@@ -76,8 +77,8 @@ public class MobsConfig {
     @Expose
     @ConfigOption(
         name = "Arachne Spawn Timer",
-        desc = "Show a timer when Arachne fragments or crystals are placed to indicate how long " +
-            "until the boss will spawn. §cTimer may be 1-2 seconds off."
+        desc = "Show a timer when Arachne fragments or crystals are placed to indicate how long until the boss will spawn.\n" +
+            "§eTimer may be 1 - 2 seconds off."
```

Comment:
adding a forceful new line here might shrink the overall text size drastically. please verify in game that the newly changed format is still good readable